PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Issue
+
+What issue is this PR targeting? In a few words, what does the PR add, change, or remove?
+
+## Tasklist
+
+- [ ] Write code & tests
+- [ ] Update relevant CHANGELOG and/or the README
+- [ ] Get a review
+- [ ] Rebase and clean up commits
+- [ ] Merge & release
```

Comment:
Are there docs about what releasing looks like?

Reply:
there is a fairly detailed section in the README about releasing - I can add a note here.

Reply:
oh, if it's there that's fine... took a glance at the readme but apparently not a good enough one. :shipit: